<?xml version="1.0"?>

<launch>
    <master auto="start"/>

    <include file="$(find qingzhou_nav)/launch/ydlidar.launch" />

    <!-- for amcl -->    
    <include file="$(find qingzhou_nav)/launch/amcl.launch" /> 

    <node name="map_server" pkg="map_server" type="map_server" args="$(find qingzhou_nav)/maps/map_7_19.yaml" output="screen"/>
    <!-- <node pkg="move_base" type="move_base" respawn="false" name="move_base" > -->
    <node name="move_base" pkg="hybrid_astar_planner" type="test_planner" output="screen">
        <rosparam file="$(find qingzhou_nav)/config/l1_local_costmap_params.yaml" command="load" />
        <rosparam file="$(find qingzhou_nav)/config/l1_global_costmap_params.yaml" command="load" />
        <rosparam file="$(find qingzhou_nav)/launch/includes/move_base_params.yaml" command="load"/>
        <param name="base_global_planner" value="global_planner/GlobalPlanner" /><!--这个标志是选择使用特定的全局规划器-->
        <!-- <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS" /> -->
        <param name="use_hybrid_astar" value="true" />
    </node>


    <node pkg="qingzhou_nav" type="L1_controller_v3" respawn="false" name="L1_controller_v3">       
        <!-- L1 -->
        <rosparam file=	"$(find qingzhou_nav)/config/l1_controller_v3_param.yaml" command="load" />
    </node>

	
</launch>
